src/yarpWholeBodyStates.cpp

```diff
@@ -193,6 +193,7 @@ bool yarpWholeBodyStates::loadCouplingsFromConfigurationFile()
 bool yarpWholeBodyStates::configureFloatingBaseStateEstimator()
 {
     yarp::os::Bottle & state_opt_bot = wbi_yarp_properties.findGroup("WBI_STATE_OPTIONS");
+    std::cerr << "\n\n" << state_opt_bot.toString() << "\n\n";
```

Comment:
ops.. it was a previous modification that I did not checkout before I created this branch